src/Connectors/test/Connector.EFCore.Test/MySqlDbContextOptionsExtensionsTest.cs

```diff
@@ -80,9 +78,13 @@ public void UseMySql_ThrowsIfServiceNameNull()
         Assert.Contains(nameof(serviceName), ex4.Message);
     }
 
-    [Fact]
-    public void AddDbContext_NoVCAPs_AddsDbContext_WithMySqlConnection()
+    [Theory]
+    [InlineData(new[]{"Pomelo.EntityFrameworkCore.MySql"}, typeof(MySqlConnector.MySqlConnection))]
+    [InlineData(new[]{"MySql.EntityFrameworkCore", "MySql.Data.EntityFrameworkCore"}, typeof(MySqlDataConnectionAlias))]
+    public void AddDbContext_NoVCAPs_AddsDbContext_WithMySqlConnection(string[] efAssemblies, Type mySqlConnection)
```

Comment:
`Type mySqlConnection` parameter: Its common practice to suffix identifier names that represent a `Type`, `MethodInfo`, `PropertyInfo` etc, to distinguish them from instances. So I suggest to rename this to `Type mySqlConnectionType`, which makes the line `Assert.True(con.GetType() == mySqlConnection);` not look like a bug.